src/java/org/apache/cassandra/index/sai/disk/format/IndexComponent.java

```diff
@@ -50,6 +50,11 @@ public enum IndexComponent
      */
     TERMS_DATA("TermsData"),
 
+    /**
+     * Product Quantization store used to store compressed vectors for the vector index
+     */
+    PQ("PQ"),
```

Comment:
Maybe `PQ` is not the most friendly nor easy to investigate name when looking at the sstable files without context. We could extend it so it looks like the other components:
```suggestion
    PRODUCT_QUANTIZATION("ProductQuantization"),
```

Reply:
That's an alternative but it is describing how the vectors are stored rather than what is being stored in the component. How about we just call it `VECTORS`?